docs/readmes/orc8r/dev_rest_api_auth.md

```diff
@@ -0,0 +1,136 @@
+---
+id: dev_token_security
+title: REST API Auth
+hide_title: true
+---
+# REST API Auth
+
+## Motivation
+
+Orc8r’s current northbound security currently uses operator-managed x509 client certificates.
+For more information on the current Orc8r current security architecture,
+read [Security Overview](./architecture_security.md#application-layer-access-control)
+for a general overview.
+
+To improve our security measures and to allow for more fine-grained control
+over resource access, we added tokens that must be sent along with each REST
+API request in the header.
+
+## Overview
+
+![rest_api_auth_overview](assets/orc8r/rest_api_auth_overview.png)
+For Orc8r’s northbound interface security, all REST API requests must include a
+client certificate and a `<username>:<token>` in the Basic Auth header.
+
+All users need to do the following to access REST API resources:
+
+1. Login with their username and password through the `/login` endpoint
+   ![rest_api_auth_1](assets/orc8r/rest_api_auth_1.png)
+2. Copy the token from the response body
+   ![rest_api_auth_2](assets/orc8r/rest_api_auth_2.png)
+3. Authenticate with their username and token (in lieu of the password field) through the Swagger Authorize tool on the top right corner
+   ![rest_api_auth_3](assets/orc8r/rest_api_auth_3.png)
+   ![rest_api_auth_4](assets/orc8r/rest_api_auth_4.png)
+
+## Bootstrapping Admin Token
+
+The initial admin user and its token can be generated via the accessc CLI.
+Exec into your docker container by running `docker-compose exec controller bash`
+in the `${MAGMA_ROOT}/orc8r/cloud/docker`, and
```

Comment:
"Exec into docker container" is only required step for docker-compose deployments. Prod deployments you'd exec into any controller pod, e.g. using k9s. So we can just say "Exec into your Orc8r application container, then run the command..."